target/linux/ramips/image/mt7621.mk

```diff
@@ -959,6 +959,23 @@ define Device/xiaomi_redmi-router-ac2100
 endef
 TARGET_DEVICES += xiaomi_redmi-router-ac2100
 
+define Device/xiaomi_mi-router-ac2100
```

Comment:
Should be moved, so alphabetic sorting is kept.

Reply:
@adrianschmutzler How exactly it should be? Before redmi-router-ac2100 or before mir3g??

Reply:
I updated to mi before redmi. Probably you mean this.